roles/1-prep/templates/iiab-expand-rootfs

```diff
@@ -34,6 +31,8 @@ if [ -f /.expand-rootfs ] || [ -f /.resize-rootfs ]; then
         growpart $ROOT_DEV $ROOT_PART_NUM
         resize2fs $ROOT_PART
 
+        # 2022-03-15: Legacy code below for microSD cards but NOT USB boot disks
+
```

Comment:
The use of /dev/sda is not just a 'usb' boot disk but any boot device that uses sdX naming, this should work nicely with 'normal' spinning disks also. (think older sata drives)